analyses/0007-porter.md

```diff
@@ -22,6 +12,14 @@ Prepared by:
 
 Date: 2023-04-07
 
+Resources:
+
+- [Meeting notes](https://docs.google.com/document/d/12OGtSaUtlc7OA_iPnUvmVaiKg8yM7_QBzFnMgoHvnLw?no-link-check)
```

Comment:
@thisisobate @nate-double-u: for some reason the [Google doc became inaccessible](https://github.com/cncf/techdocs/actions/runs/13551833642/job/37877016171?pr=295), so I'm marking it with `no-link-check`.

Reply:
Sounds good, i'll keep `?no-link-check` in my back pocket for other links like this. Weird it's not accessible now tho.

Reply:
At a minimum, it should probably be made public.